shared/CORS_list.go

```diff
@@ -0,0 +1,8 @@
+// Copyright 2020 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package shared
+
+// CORSList is a list of trusted third-party origins.
+var CORSList = []string{"https://jgraham.github.io"}
```

Comment:
It probably makes more sense to have it in `api/routes.go`. CORS is specific to webapp APIs and I don't expect this to be "shared".

Reply:
Right, but this `CORSList` is shared by `webapp` and `api`. By having it in `api/routes.go`, `api` dependency is introduced in `webapp`. What if I move this login/status endpoint to `api`?